resource_activity/models/resource.py

```diff
@@ -47,6 +49,45 @@ class ResourceLocation(models.Model):
     guides = fields.One2many('res.partner', 'resource_location_guide', domain=[('is_guide', '=', True)], string="Guides")
     trainers = fields.One2many('res.partner', 'resource_location_trainer', domain=[('is_trainer', '=', True)], string="Trainers")
     opening_hours_ids = fields.Many2many('activity.opening.hours', string="Opening Hours")
+    line_ids = fields.One2many(

```

Comment:
`term_ids` would have been a better name.

Reply:
+1

Reply:
will do :)

Reply:
changed to `terms_ids`